stdeb.cfg

```diff
@@ -1,13 +1,10 @@
 [DEFAULT]
 ; release with a high debinc to avoid conflict with upstream debian of the same release version
 Debian-Version: 100
-Depends: python-yaml, python-empy, python-argparse, python-packaging, python-rosdep (>= 0.15.0), python-rosdistro (>= 0.8.0), python-vcstools (>= 0.1.22), python-setuptools, python-catkin-pkg (>= 0.4.3)
 Depends3: python3-yaml, python3-empy, python3-packaging, python3-rosdep (>= 0.15.0), python3-rosdistro (>= 0.8.0), python3-vcstools (>= 0.1.22), python3-setuptools, python3-catkin-pkg (>= 0.4.3)
 Conflicts: python3-bloom
```

Comment:
We could consider dropping the `Conflicts3:` as well, but we should at least drop the Python 2 `Conflicts: ` entry.

```suggestion
```